### PR TITLE
Interpolation Section

### PR DESCRIPTION
**Pull request by varughese**
Attempting to finish #13

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -323,12 +323,80 @@ Templates in React are written inside the JavaScript file using the [JSX languag
...
-> TODO
+### AngularJS
+Interpolation is the process of data-binding values on the AngularJS `scope` to values in the html. You can read more on the [official documentation](https://docs.angularjs.org/guide/interpolation):
+
+Let's say we have a value `heroImageUrl` on our scope that is defined as `superman.jpg`.
+
+The following HTML
+
+```html
+<p><img src="images/{{heroImageUrl}}"> is the <i>interpolated</i> image.</p>
```
Comment (feimosi) [inline README.md]: That's actually an anti-pattern. People shouldn't use interpolation in the `src` attribute. There's `ng-src` for that:
https://docs.angularjs.org/api/ng/directive/ngSrc

File: README.md [hunk 2]
```diff
@@ -323,12 +323,80 @@ Templates in React are written inside the JavaScript file using the [JSX languag
...
+```html
+<label>Enter name: <input type="text" ng-model="name"></label><br>
+Hello <span ng-bind="name"></span>!
+```
+In this example, whatever is typed into the input will be placed on the `name` variable on the scope. The `ng-bind` will cause the content of the span to be updated to be the value of `name`. whenever the input changed. See the full example [here](https://docs.angularjs.org/api/ng/directive/ngBind).
+
+### Angular
+Angular is similar to AngularJS. You can read more on the [official documentation](https://angular.io/guide/template-syntax#interpolation----).
+
+`{color}` Will still interpolate to `red`.
```
Comment (feimosi) [inline README.md]: Should be `{{ color }}`.

**Review by feimosi — CHANGES_REQUESTED** [review]
Thanks! Take a look at my comments please.

**Comment by varughese** [comment]
Alright I made those changes

**Review by feimosi — APPROVED** [review]

**Comment by feimosi** [comment]
Looks good, thanks!

**Pull request merged**